src/hooks/use-launch-notification.ts

```diff
@@ -0,0 +1,118 @@
+/**
```

Comment:
I reckon use-computation-notification would be more appropriate to refer to loadflow, security analysis etc...